sdk/src/main/java/com/google/cloud/dataflow/sdk/options/DataflowPipelineWorkerPoolOptions.java

```diff
@@ -254,4 +254,16 @@ public String getTeardownPolicyName() {
       + "https://cloud.google.com/compute/docs/reference/latest/diskTypes")
   String getWorkerDiskType();
   void setWorkerDiskType(String value);
+
+  /**
+   * Specifies whether worker pools should be started with public IP addresses.
+   *
+   * <p>WARNING: This feature is experimental.  You must be whitelisted to use it.
+   */
+  @Description("Specifies whether worker pools should be started with public IP addresses. WARNING:"
+    + "This feature is experimental. You must be whitelisted to use it.")
+  @Experimental
+  @JsonIgnore
+  Boolean getUsePublicIps();
```

Comment:
I notice that the option you send on the wire is a string. Do you anticipate future policies other than public/private? It just seems counter-intuitive to me that these are mismatched.